lib/guestDrawer.js

```diff
@@ -0,0 +1,106 @@
+const { DRAWER_ID } = require('./constants')
+const styles = require('./styles/drawer.module.scss');
+
+/**
+ * This adds a drawer that is used to display vital statistics
+ */
+module.exports = (api) => {
+
+	const { h, thunk, DrawerToggle, Tooltip, DrawerHelp } = api.render;
+	const { array, value } = api.observ;
+	const { displayText } = api;
+
+	const counterState = value(1);
+	const calculationsState = array([]);
+
+	const state = api.observ.state({
+		input: value(undefined),
+		output: value(undefined)
+	});
+
+	// Load our WebAssembly API
+	const wasmApi = WebAssembly.instantiateStreaming(fetch('https://plugins.coviu.com/examples/wasm/test.wasm')).then(({ instance, module }) => {
```

Comment:
what's this for?

Reply:
This is a Webassembly to return some random numbers. This can be done via webhook in the next card

Reply:
undersood, thanks